lib/github/commands/rest2html

```diff
@@ -162,7 +165,11 @@ def main():
     except IOError:  # given filename could not be found
         return ''
     except IndexError:  # no filename given
-        text = sys.stdin.read()
+        if sys.version_info[0] < 3: # python 3
+            input_stream = io.TextIOWrapper(sys.stdin.buffer, encoding='utf-8')
+            text = input_stream.read()
+        else: # python 2
+            text = sys.stdin.read()
```

Comment:
note that i've submitted that one to the python-future website to see what the best idiom would be according to them: https://github.com/PythonCharmers/python-future/issues/241